dist/stat.py

```diff
@@ -6,15 +6,15 @@
 from operator import attrgetter
 
 # Read the source files.
-from stat_data import groups, dsrc_stats, connection_stats, conn_dsrc_stats, join_stats, \
+from stat_data import groups, dsrc_stats, conn_stats, conn_dsrc_stats, join_stats, \
     session_stats
 
-connection_statistics = connection_stats
-connection_statistics.extend(conn_dsrc_stats)
-connection_statistics = sorted(connection_statistics, key=attrgetter('desc'))
-dsrc_statistics = dsrc_stats
-dsrc_statistics.extend(conn_dsrc_stats)
-dsrc_statistics = sorted(dsrc_statistics, key=attrgetter('desc'))
+sorted_conn_stats = conn_stats
+sorted_conn_stats.extend(conn_dsrc_stats)
+sorted_conn_stats = sorted(sorted_conn_stats, key=attrgetter('desc'))
```

Comment:
Please add a comment as to why we're sorting them, something along the lines of "statistic categories need to be sorted otherwise the wiredtiger statistics json will be invalid."

Reply:
3a87d4d